LastWork(newWork)/Views/evaluations/evaluations.js

```diff
@@ -1,156 +1,151 @@
-function EvaluationsTableHeader(option) {
-    const headings = [];
-    option.headings.forEach((el) => {
-        const generatedEl = `<th class="details_button">${el}</th>`;
-        headings.push(generatedEl);
-    });
-    const headingsEl = headings.join('');
+// function EvaluationsTableHeader(option) {
```

Comment:
If it's commented out and no longer needed, consider removing altogether for a bit of cleanup.